src/app/_blocks/HubContentGrid/NavBar/Buttons/index.tsx

```diff
@@ -20,7 +20,7 @@ export function AllContent({ fill = 'var(--dark-rock-800)', textColor = 'var(--s
         d="M158.75 60.5H156.855C150.225 60.5 143.866 57.8661 139.178 53.1777L94.3223 8.32233C89.6339 3.63392 83.2751 1 76.6447 1H26C12.1929 1 1 12.1929 1 26V60.5"
         stroke="#403F4C"
       />
-      <AllIcon x="35px" y="15px" width="20" height="20" stroke={textColor} />
+      <AllIcon x="35px" y="15px" width="20" height="20" color={textColor} />
```

Comment:
I had no idea that it was ok to place an svg inside another svg, but it is 🤔 TIL

Reply:
SVGception 😵 
(took me a while to make things work out)